RecoLocalTracker/Phase2TrackerRecHits/plugins/Phase2TrackerRecHits.cc

```diff
@@ -63,8 +63,8 @@ void Phase2TrackerRecHits::produce(edm::StreamID sid, edm::Event& event, const e
   eventSetup.get< TrackerDigiGeometryRecord >().get(geomHandle);
   const TrackerGeometry* tkGeom(&(*geomHandle));
 
-  edm::ESHandle< TrackerTopology > tTopoHandle;
-  eventSetup.get< IdealGeometryRecord >().get(tTopoHandle);
+  //edm::ESHandle< TrackerTopology > tTopoHandle;
+  //eventSetup.get< IdealGeometryRecord >().get(tTopoHandle);
```

Comment:
It's a leftover from stripping down from something existing. I can remove it, and put it back later if it turns out later that it'd be needed.
Since I'm a total newbie, a stupid question: how does this work? I push this change and make a new pull request?